### ISSUE TITLE
missing link in preview opens tiddler in edit mode

### ISSUE THREAD

**Issue by alexhough** [Jermolene/TiddlyWiki5#3119]
Could be an edit mode where a link clicked on in preview always opens a draft?

With tiddlers opening above the tiddler being edited, a pleasing zig zag workflow could develop, see below

![screen shot 2016-05-09 at 20 50 22](https://user-images.githubusercontent.com/245512/36214748-3063fdd0-11a2-11e8-8279-419d96dc4e28.png)

https://groups.google.com/d/msg/tiddlywiki/a3m2uJq-aig/L9ch6DCaBAAJ

**Comment by Jermolene** [comment]
Good idea @alexhough. Since the introduction of the "actions" attribute for the `<$linkcatcher>` widget I think this would be fairly straightforward to do if anyone is interested in trying an implementation?
